src/getHumanAge.test.js

```diff
@@ -7,4 +7,18 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return 0 human years for cat or dog less than 15 years', () => {
+    expect(getHumanAge(10, 7)).toEqual([0, 0]);
```

Comment:
is should be only one expect() call in test() block. You can create more test() blocks and put them there